src/lib.rs

```diff
@@ -32,6 +32,11 @@
 //!
 //! - `hexx = { version = "0.7", features = ["packed"] }`
 //!
+//! `hexx` supports [Bevy Reflection](https://docs.rs/bevy_reflect/latest/bevy_reflect) through the
+//! `bevy_reflect` feature. To enable it add the following line to your `Cargo.toml`:
+//!
+//! - `hexx = { version = "0.8", features = ["bevy_reflect"] }`
```

Comment:
```suggestion
//! - `hexx = { version = "0.7", features = ["bevy_reflect"] }`
```

This feature might be released for 0.7.1